recipe/build.sh

```diff
@@ -24,4 +24,10 @@ if [ "$(uname)" == "Linux" ]; then
   make check \
     || { cat itaps/imesh/test-suite.log; exit 1; }
 fi
+
+if [[ -n "$tempest" && "$tempest" != "notempest" && -n "$mpi" && "$mpi" != "nompi" ]]; then
+  make -C test imoab_remapping && test/imoab_remapping
+  make -C test/parallel imoab_coupler && mpiexec -n 2 test/parallel/imoab_coupler
```

Comment:
@vijaysm, are these already run in `make check` above?  I think so, so I think this may be redundant.

Reply:
Yes `make check` builds and runs these and all other tests as well. Not sure if we want the full blown check for verification.

Reply:
Yes, we want to full check and it's already happening above.  I'm going to remove these redundant tests, then.